### ISSUE TITLE
[CAIA A11y Research - PILOT]: MHV Medical Records (August 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#91078]
> [!IMPORTANT]
> - **TICKET STATUS:** `COMPLETED`
> - **PILOT DATE:**  08/20/2024 at 1:00 p.m. ET 
> - **TYPES OF ASSISTIVE TECH:** `Type used in pilot`
> - **PROTOTYPE**: [CodeSpace](https://glorious-sniffle-rqjx676prgx2wjqx-3001.app.github.dev/my-health/medical-records/?postLogin=true) (choose "yes" to trust inquiry, if prompted)

# CAIA A11y Support
This is a child ticket of the research ticket  #90780 - created to track work, completed from this pilot session. 

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

@rileyorr  

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: MHV on VA.gov - Medical Records
- OCTO-DE product owner: Marci McGuire
- Product manager: Anne Costello Kennedy
- Designer:
- FE engineer:
- Accessibility specialist: Riley Orr
- Product/team Slack channel: #mhv-medical-records

</details>

## Description

<details><summary>Toggle to view the description</summary>

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a VA employee that is a screen reader user
- **User**: A VA employee that is a screen reader user 
- **Assistive Tech**: The CAIA A11Y team member will follow along with the session and take notes while providing support to the participant as needed. 

### Pilot Session
**Date:** August 20, 2024

#### Pilot Resources
- Internal CAIA Notes Doc `Link'
- [Prototype `Link`](https://glorious-sniffle-rqjx676prgx2wjqx-3001.app.github.dev/my-health/medical-records/?postLogin=true)
- GitHub Resources
    - [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/research-plan.md#timeline--screen-reader-users)
    - [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/conversation-guide.md)
    - [Assistive Technology Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md#assistive-technology-pilot-guide)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Research Goals

- Identify any usability issues Veterans may experience in accessing their Care Summaries and Notes, Lab and Test Results, and Health Conditions.
- Understand Veterans’ sensemaking of medical terminology in Medical Records.
- Understand Veterans’ needs and expectations within Medical Records.

### Research Questions

- Can Veterans find the information they need to manage their healthcare related to:
   -  Care Summaries and Notes
   - Lab and Test Results
   - Health Conditions
- How do Veterans make sense of the terminology in their medical records?
- What goals are Veterans hoping to accomplish within Medical Records?
- What are Veterans’ expectations and use cases for printing/downloading Medical Records?
- What are Veterans’ perceptions of the new experience in which they view their records on screen?

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** @rileyorr
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:** August 20 1:00 p.m. ET
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled `add to CAIA calendar`
- [x] **Review research** - Riley
- [x] Request AT device and tech to be used in sessions Screen Readers (8) Magnifiers/Zoom (5) Text-to-Speech (3)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] Review Goals in Research Plan
- [x] **Google Doc Template:** [Plan/create prep doc template](https://docs.google.com/document/d/1uZsaSNyaFThE7-sSkx-5hUXC8qMq2ZYu9ofxwlXh2SU/edit#heading=h.7b53y4dqihzk)
- [x] **Google Doc:** [Pilot Prep Doc](https://docs.google.com/document/d/19TzkA5jjhdtYTUMcLj5FBkwqGr1_-rVc5I7JzsBnY5s/edit?usp=sharing)
- [x] **Review prototype** - [Prototype](https://glorious-sniffle-rqjx676prgx2wjqx-3001.app.github.dev/my-health/medical-records/?postLogin=true)
- [x] **Run pilot** with VFS Researcher - Riley
- [x] Connect to primary research ticket - Sara
```

**Comment by rileyorr** [comment]
Hi @AnneCostello! Thanks for having me in yesterday's pilot. I've dropped my takeaways below but they're also available in [the pilot notes document](https://docs.google.com/document/d/19TzkA5jjhdtYTUMcLj5FBkwqGr1_-rVc5I7JzsBnY5s/edit?usp=sharing). Looking forward to helping with next week's sessions! Lmk if you have any questions. 

**Task: Viewing lab and test results** 
 - Navigates to the left menu nav and selects ‘Lab and test results’
![Screenshot from the prototype showing two items from the list of lab and test results. The first item is for complete blood count on April 1, 2022, 9:00 a.m. ordered by Beth M. Smith and the second item is for complete blood count on March 10, 2022, 9:30 a.m. ordered by Beth M. Smith. There is a red focus indicator wrapping around the contents of the first complete blood count item and the title of the second complete blood count item.](https://github.com/user-attachments/assets/3cbbab9e-1bf3-410a-bedd-0b342225ed33)

- The focus coupling on headers from multiple cards - what is the screen reader reading out? (SC audio isn’t being shared)
   - The participant was asked at the end of the session to navigate back to this page to test the cards; the correct title is reading out 
   - Hypothesis: For some reason the focus size is this size on all the cards. I noticed it remained the same size even as the participant navigated to the last card on the page (I was unable to get a screenshot during the session). There could be hidden content within the focus.

**Task: Information about general health conditions and diagnoses** 
- Expects a glossary before navigating into a specific condition (navigated into hypertension) 

**Task: View provider notes from past visit** 
- Struggled to find the HT Monthly Monitor Note, needed to be prompted by the moderator 
   - They had tabbed through the list, I wonder if the screen reader was reading the title in the same way the moderator was saying it? For example, some screen readers aren’t great at natively reading out individual letters so it may have combined the H and T to try to form a word. Would need to double check what's read out and if there are aria tags being used.
   - Participant suggested putting “Home Healthcare Monthly” or something. “You’re getting VA jargon”
      - “You don’t even need the word ‘tele’, just ‘health’

**Task: Determining hospital stay lengths**
- Was using context of the individual care summaries to try to determine how many hospital stays there were are 
   - Ex. anesthesia was a hint that Pat was in the hospital
- Determines 2 hospital stays due to the 2 discharge summaries 
- “Discharge summary was the first obvious one” 
- When asked to rank the experience: “If information was reversed in the way the information was displayed it would be a 5. It’s a 3 right now. If I wasn’t a patient person I would get irritated with it.” 
   - “Tell me what I don’t already know - give me that stuff first” 

**Task: COVID test results**
- Understands ‘Not detected’ on the second test reviewed as meaning that Pat didn't have COVID
- When going through blood count results, noted that it would be helpful for the reference range to read out the context of what it relates to rather than just reading out the numbers
- Mentions again that they don’t know what acronyms mean and would want them defined or in a glossary 
- When asked if Pat has a cholesterol problem, they would expect to have found it in the complete blood count but wasn’t sure what to look for because of all the acronyms 
   - The moderator prompts them to the ‘Lipid panel’ card and when asked if that title has any meaning the participant says no 
- Rated the experience 2 or 3
   - Easier paths or relabeling to “not jargon” would bump it to a 5
